subsys/net/pkt_filter/Kconfig

```diff
@@ -19,4 +19,19 @@ module-help = Enables packet filter output debug messages
 source "subsys/net/Kconfig.template.log_config.net"
 endif # NET_PKT_FILTER
 
+config NET_FILTER
```

Comment:
I'm actually a bit uncertain, whether NET_FILTER isn't too broad of a name for the module. Maybe NET_IP_FILTER? I'm open for discussion though.

Reply:
In linux it's jut netfilter, NET_IP_FILTER sounds good when we look to current implementation (now I'm focusing on the IP filtering) but it can be used also for other purposes.
What do you think about NET_HOOKS_FILTER?

Reply:
`NET_HOOKS_FILTER` sounds a bit awkward to me, I'd prefer to stick to `NET_FILTER` if we don't want to use IP. Although it may seem a bit ambiguous, given there's `NET_PKT_FILTER` already.